### ISSUE TITLE
[Bug] Roslynator CLI 0.13.1.0 raises various CS errors when using extension members

### ISSUE THREAD

**Issue by cmpbedes** [JosefPihrt/Roslynator#1827]
### Product and version

Roslynator.Analyzers, Roslynator.CodeAnalysis.Analyzers, Roslynator.Formatting.Analyzers - all 14.6.1, Rosylnator.DotNet.CLI 0.13.1.0

### Rule or diagnostic ID

CS0103, CS0708, CS0710, CS1022, CS1513, CS1520

### Steps to reproduce

1. install Roslynator.DotNet.CLI 0.13.1.0: `dotnet tool install --global Roslynator.DotNet.CLI --version 0.13.1.0`
2. Extract attached repro
3. run `roslnator analyze ExtensionMemberTest.slnx

[ExtensionMemberTest.zip](https://github.com/user-attachments/files/31303730/ExtensionMemberTest.zip)

### Actual behavior
Raises CS0103, CS0708, CS0710, CS1022, CS1513, CS1520 diagnostics

### Expected behavior

No diagnostics reported

### Exception messages and stack traces

```shell
roslynator analyze ExtensionMemberTest.slnx
Loading solution 'D:\PROJECTS\github\public\ExtensionMemberTest\ExtensionMemberTest.slnx'...
Analyze solution 'D:\PROJECTS\github\public\ExtensionMemberTest\ExtensionMemberTest.slnx'
Analyze 'ExtensionMemberTest' 1/1
  FooExtensions.cs(9,31): error CS0103: The name 'foo' does not exist in the current context
  FooExtensions.cs(7,21): error CS0708: 'PrintHelloWorld': cannot declare instance members in a static class
  FooExtensions.cs(5,5): error CS0710: Static classes cannot have instance constructors
  FooExtensions.cs(12,1): error CS1022: Type or namespace definition, or end-of-file expected
  FooExtensions.cs(6,6): error CS1513: } expected
  FooExtensions.cs(5,5): error CS1520: Method must have a return type
```

**Comment by josefpihrt** [comment]
New version of cli tool which references Roslyn 5 will be released very soon. That should fix the issue.

**Comment by josefpihrt** [comment]
New version has been released: https://www.nuget.org/packages/Roslynator.DotNet.Cli/1.0.0
